src/plugins/lanplus/lanplus.c

```diff
@@ -1165,14 +1158,11 @@ read_session_data(
  *              the end of the session when this function exits.
  *  param s      holds our session state
  */
-void
-read_session_data_v2x(
-					  struct ipmi_rs      * rsp,
-					  int                 * offset)
+void read_session_data_v2x(struct ipmi_rs *rsp, int *offset)
 {
 	rsp->session.authtype = rsp->data[(*offset)++];
 
-	rsp->session.bEncrypted     = (rsp->data[*offset] & 0x80 ? 1 : 0);
+	rsp->session.bEncrypted = (rsp->data[*offset] & 0x80 ? 1 : 0);
 	rsp->session.bAuthenticated = (rsp->data[*offset] & 0x40 ? 1 : 0);
```

Comment:
Magic masks this time... Definitely needs a separate commit to get rid of 'em all.